.github/workflows/ci.yml

```diff
@@ -118,7 +118,7 @@ jobs:
         run: |
           echo "${{ github.workspace }}/external/dlls-${{ matrix.target.cpu }}" >> $GITHUB_PATH
 
-      - name: Derive environment variables
```

Comment:
these github CI files are 99% the same across all projects - let's leave them like they are so that changes can easily be applied to all projects (or apply change to all projects) - also, looks like i386 build gets broken.

If there are things that should be common, maybe we can add them to nimbus-build-system/build.sh that gets downloaded by every build instead